Acrolinx.Sidebar.SDK/Utilities/DllUtil/DllUtil.h

```diff
@@ -20,6 +20,8 @@ namespace Acrolinx_Sdk_Sidebar_Util
         static BOOL GetDllInfo(CString& id, CString& name, CString& version);
         static CString GetAppName(void);
         static CString GetLastErrorAsString(void);
+        static HINSTANCE GetDllHandle(void);
```

Comment:
check access specifier for this method

Reply:
We need this to be public as it also used in FileUtil to get resource embedded resource from DLL